MITADS-Speech/corpora_importer.py

```diff
@@ -122,9 +123,18 @@ def _maybe_extract(self,target_dir, extract_dir, archive_path):
         extracted_path = os.path.join(target_dir, extract_dir)
         if not os.path.exists(extracted_path):
             print(f"No directory {extracted_path} - extracting archive...")
-            with ZipFile(archive_path, "r") as zipobj:
-                # Extract all the contents of zip file in current directory
-                zipobj.extractall(target_dir)
+
+            ##check file if zip or tar
+            if(archive_path.endswith('.zip')):
+                ##extraxt zip file
+                with ZipFile(archive_path, "r") as zipobj:
+                    # Extract all the contents of zip file in current directory
+                    zipobj.extractall(target_dir)
+            else:
+                ##extract other gzip, bz2 and lzma
+                tar = tarfile.open(archive_path)
+                tar.extractall(extracted_path)
```

Comment:
child folder with same name of its parent (eg: origin/mls_italian/mls_italian)
```suggestion
                tar.extractall(target_dir)
```